vyuh_cli/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+## 0.0.1
+
+- First release with a set of core commands
+  - `vyuh create project`
+  - `vyuh create feature`
+  - `vyuh create feature-sanity-schema`
```

Comment:
we should just call it `schema` and then accept the CMS (as a parameter) later when we support multiple cms-es

Reply:
Changed to "schema," the "cms" option has been added, with the default  CMS as "sanity."